package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
+    "chart.js": "^4.2.1"
```

Comment:
chart package should be used via CDN, not NPM. there is no reason to use NPM in this project.